test/gurka/test_gtfs.cc

```diff
@@ -862,6 +862,7 @@ TEST(GtfsExample, route_trip1) {
   // wrap around the next month if it's getting critical
   if (tmrw_int > 27) {
     auto new_month = std::to_string(std::stoul(std::string(&req_time[5], &req_time[7])) + 1);
```

Comment:
and when its december? shouldnt we let the date time library handle this. it has support for addition and subtraction but requires a little bit more work and temporary values of course..

Reply:
argh.. right.. I'll write an issue about it. I'm sure I'll get to it before 2024;)